src/main/java/us/dot/its/jpo/ode/aws/depositor/AwsDepositor.java

```diff
@@ -176,7 +185,7 @@ public void run(String[] args) throws Exception {
 				boolean gotMessages = false;
 
 				while (true) {
-					ConsumerRecords<String, String> records = stringConsumer.poll(CONSUMER_POLL_TIMEOUT_MS);
+					ConsumerRecords<String, String> records = stringConsumer.poll(Duration.ofMillis(CONSUMER_POLL_TIMEOUT_MS));
```

Comment:
Why was this necessary?

Reply:
The poll(long) method was deprecated - I updated this to use a duration instead so that there would no longer be a deprecation warning.